<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="graphwidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1161</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="PID_group">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>511</width>
      <height>101</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>PID Control</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLCDNumber" name="v_out_disp">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>45</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.255000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="p_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>45</y>
       <width>48</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="dt_label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>dt</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>84</y>
       <width>20</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="k_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>45</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="k_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>18</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dt_SpinBox">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>45</y>
       <width>39</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="i_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>45</y>
       <width>48</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>20.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="d_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>45</y>
       <width>48</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>20.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="PID_checkBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dt_label_2">
     <property name="geometry">
      <rect>
       <x>447</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DC, %</string>
     </property>
    </widget>
    <widget class="QLabel" name="d_label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>18</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QLabel" name="i_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QLabel" name="p_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="k_label_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PID on</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Plot Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="inf_label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>PID Out</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="PIDout_checkBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="win_SpinBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="inf_checkBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="int_SpinBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="int_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>15</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Interval(ms)</string>
     </property>
    </widget>
    <widget class="QLabel" name="inf_label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Infinite</string>
     </property>
    </widget>
    <widget class="QLabel" name="win_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Window (s)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>72</y>
       <width>81</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_button">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>81</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load_button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Laser Control</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="laser_low_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser Low</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sp_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximum">
      <double>4.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set Point</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="shootBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Shooting</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
